### PR TITLE
Update drupal-drush-command-line-utility.md

### PR DESCRIPTION
**Pull request by ttrowell**
Steps for adding a custom Drush command

@bmackinney I think this is where you intended the section to be added.  @nataliejeremy please review my formatting!  Thank you

### CODE REVIEW

File: source/docs/articles/local/drupal-drush-command-line-utility.md [hunk 1]
```diff
@@ -193,6 +193,22 @@ As an alternative to `drush sql-sync` you can use `drush sql-dump` instead.
...
+While we have the full spectrum of drush core already available for your use, you may want to add a command that you regularly use, for instance, [drush search and replace (sar)](https://www.drupal.org/project/sar). 
+  
+1. Put the site in Git mode.
+2. Clone locally.
+3. Create a “drush” folder in the Drupal root.
+4. Add the “sar” drush command to the “drush” folder.
+5. Commit drush/sar.
+6. Push your code up to master. 
+7. Deploy to Test and Live.
+8. Download your Pantheon [drush aliases](/docs/articles/local/drupal-drush-command-line-utility/#list-available-site-aliases).
```
Comment (nataliejeremy) [inline source/docs/articles/local/drupal-drush-command-line-utility.md]: @ttrowell I added this link. Is this what you wanted to reference?

**Comment by nataliejeremy** [comment]
@ttrowell Thanks! Just saw your blog post and updated with the right link.

File: source/docs/articles/local/drupal-drush-command-line-utility.md [hunk 2]
```diff
@@ -193,6 +193,22 @@ As an alternative to `drush sql-sync` you can use `drush sql-dump` instead.
...
+2. Clone locally.
+3. Create a “drush” folder in the Drupal root.
+4. Add the “sar” drush command to the “drush” folder.
+5. Commit drush/sar.
+6. Push your code up to master. 
+7. Deploy to Test and Live.
+8. Download your Pantheon [drush aliases](https://pantheon.io/blog/drush-aliases-available).
+9. Clear your drush cache on each environment. (example: “drush @pantheon.<site-name>.devcc drush”)
+
+If you have successfully set up [Terminus](https://pantheon.io/docs/articles/local/cli/), the Pantheon CLI, you can get your drush aliases by using “terminus sites aliases”. At this point, you are able to start using the drush command you added.  Drush 5 is the default version for newly created sites on Pantheon.  However, to have strict control over your drush site aliases, see [Fix Up Drush Site Aliases with a Policy File](https://pantheon.io/blog/fix-drush-site-aliases-policy-file).
```
Comment (bmackinney) [inline source/docs/articles/local/drupal-drush-command-line-utility.md]: Let's apply the same principle here - get the "how to" out of the blog and add it to this doc.

Comment (bmackinney) [inline source/docs/articles/local/drupal-drush-command-line-utility.md]: Also we can make both URLs in this line relative.

**Comment by rachelwhitton** [comment]
@bmackinney instructions from the blog post added into new section

+1

**Pull request merged**